compiler/rustc_borrowck/src/region_infer/mod.rs

```diff
@@ -335,7 +336,7 @@ fn sccs_info<'tcx>(infcx: &BorrowckInferCtxt<'tcx>, sccs: &ConstraintSccs) {
 
     let mut reg_vars_to_origins_str = "region variables to origins:\n".to_string();
     for (reg_var, origin) in var_to_origin_sorted.into_iter() {
-        reg_vars_to_origins_str.push_str(&format!("{reg_var:?}: {origin:?}\n"));
+        writeln!(reg_vars_to_origins_str, "{reg_var:?}: {origin:?}").unwrap();
     }
     debug!("{}", reg_vars_to_origins_str);
```

Comment:
```suggestion
    debug!("{reg_vars_to_origins_str}");
```

Mind just updating to the edition2021 syntax while you're around here?